### PR TITLE
added system counter to deal with pruning within the time map

### LINKED ISSUE #33: system time map error when frame rate not 1

### PR DESCRIPTION
**Pull request by JFRudzinski**
@ladinesa - I had to add a counter within the system loop in parse_system for the case that we are pruning the trajectory with frame_rate, so that the system_time_map is correct.

### CODE REVIEW

File: atomisticparsers/gromacs/parser.py [hunk 1]
```diff
@@ -712,12 +713,12 @@ def get_composition(children_names):
             time_step = self.log_parser.get('input_parameters', {}).get('dt', 1.0) * ureg.ps
             if sec_system.time is not None:
                 self._system_time_map[round(ureg.convert(
-                    sec_system.time.magnitude, sec_system.time.units, ureg.picosecond), 5)] = n
+                    sec_system.time.magnitude, sec_system.time.units, ureg.picosecond), 5)] = system_counter
```
Comment (ladinesa) [inline atomisticparsers/gromacs/parser.py:716]: I think we do not need system_counter, simply len(system_step_map).

**Comment by ladinesa** [comment]
Oh yes, i missed this. Please merge.

File: atomisticparsers/gromacs/parser.py (hunk 1 above)
Comment (ladinesa) [reply atomisticparsers/gromacs/parser.py:716]: please include this in next pr

Comment (JFRudzinski) [reply atomisticparsers/gromacs/parser.py:716]: oh sorry, I misunderstood, I thought you were saying nevermind below. I will change it now.

Comment (ladinesa) [reply atomisticparsers/gromacs/parser.py:716]: It's ok its minor, can go with another pr.

**Pull request merged**